White screen when launching the AppImage (Fedora 42)

### Version

0.0.47

### Operating System

Linux

### Distribution Method

AppImage (Linux)

### Describe the issue

Whether I launch the AppImage with the file explorer or via CLI, I get a white screen on the application, after the "We are back." screen, which I see.

<img width="1161" height="746" alt="Image" src="https://github.com/user-attachments/assets/39cf02dc-9b20-49a9-b492-49872bb227f1" />

### How to reproduce (Optional)

_No response_

### Expected behavior (Optional)

_No response_

### Relevant log output (Optional)

```shell
./Popcorn_Time_Nightly_0.0.47_amd64.appimage                                                            ─╯
2025-09-28T18:12:10.571969Z  INFO crates/popcorntime-tauri/src/main.rs:88: starting app version=0.0.47 name=Popcorn Time Nightly
2025-09-28T18:12:10.574221Z  INFO crates/popcorntime-session/src/authorization.rs:83: Creating new authorization broker for cb363f1c-075c-4813-86d0-0235a1e77a5e
2025-09-28T18:12:10.630375Z  INFO crates/popcorntime-session/src/jwks.rs:47: Creating JwksClient for issuer: https://accounts.popcorntime.app/.well-known/jwks.json
2025-09-28T18:12:10.696944Z  INFO crates/popcorntime-session/src/storage.rs:145: settings.json initialized
2025-09-28T18:12:10.725951Z  INFO crates/popcorntime-tauri/src/main.rs:148: Popcorn Time is ready!
2025-09-28T18:12:11.926154Z  INFO is_onboarded: crates/popcorntime-tauri/src/session.rs:38: close time.busy=9.91µs time.idle=30.2µs
2025-09-28T18:12:11.926962Z  INFO validate: crates/popcorntime-session/src/lib.rs:109: Refreshing token
2025-09-28T18:12:11.927006Z ERROR validate: crates/popcorntime-tauri/src/session.rs:31: error=Error(errors.session.invalid

Caused by:
    No refresh token found)
2025-09-28T18:12:11.927037Z  INFO validate: crates/popcorntime-tauri/src/session.rs:31: close time.busy=159µs time.idle=39.4µs
[2025-09-28][18:12:11][webview:e@tauri://localhost/assets/index-B8kKffXm.js:62:388][ERROR] [object Object]
GStreamer element appsink not found. Please install it.
GStreamer element autoaudiosink not found. Please install it

(WebKitWebProcess:401385): GLib-GObject-CRITICAL **: 20:12:21.079: invalid (NULL) pointer instance

(WebKitWebProcess:401385): GLib-GObject-CRITICAL **: 20:12:21.079: g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed
```

--- a/crates/popcorntime-tauri/tauri.conf.json
+++ b/crates/popcorntime-tauri/tauri.conf.json
@@ -26,6 +26,11 @@
     "active": true,
     "targets": "all",
     "createUpdaterArtifacts": true,
+    "linux": {
+      "appimage": {
+        "bundleMediaFramework": true
+      }
+    },
     "icon": [
       "icons/32x32.png",
       "icons/128x128.png",

